app/templates/components/feed-item.hbs

```diff
@@ -0,0 +1,16 @@
+<div class="article-preview">
+    <div class="article-meta">
+        <a href="profile.html"><img src= {{imageSrc}} /></a>
+        <div class="info">
+            <a href="" class="author">{{author}}</a>
+            <span class="date">{{date}}</span>
+        </div>
+        <button class="btn btn-outline-primary btn-sm pull-xs-right">
+            <i class="ion-heart"></i> {{hearts}}
+        </button>
+    </div>
+    <a href="" class="preview-link">
+        {{content}}
+        {{yield}}
```

Comment:
I don't think the `{{yield}}` is necessary here. Ideally, instead of passing in each property separately, we would just pass in a single `article` model instance.  I can't see using this as a block-style component.